lofty.com.website.json

```diff
@@ -82,16 +75,17 @@
       "txtConflictMatchingMode": "All"
     },
     {
-      "type": "SPFM",
+      "type": "TXT",
       "groupId": "email",
-      "host": "@",
+      "host": "mail",
       "ttl": 3600,
-      "spfRules": "include:amazonses.com"
+      "data": "v=spf1 include:amazonses.com ~all",
```

Comment:
If it's only GoDaddy I don't like in particular then affecting all other providers. Even if this is a subdomain the impact may be smaller but nevertheless not non existing. 

Better approaches:
- special group for GoDaddy variant
- let GoDaddy transform the template during onboarding
- special separate template